server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionCoordinator.java

```diff
@@ -150,6 +154,9 @@ protected CompactionCoordinator(ServerOpts opts, String[] args, AccumuloConfigur
     printStartupMsg();
     startCompactionCleaner(schedExecutor);
     startRunningCleaner(schedExecutor);
+    // TODO use refresh mechanism to avoid RPCs blocking on getting this count when it expires
```

Comment:
No, that todo is probably an unneeded optimization that is not supported by any data.  I added some logging and removed the todo in 51770be.  In testing the log message locally noticed it would take a few ms the first count and 0ms for the 2nd count because its reading from zoocache. Thinking the logging can help if the counts ever turn out to be a problem.